twilight-lavalink/src/node.rs

```diff
@@ -488,15 +503,20 @@ impl Connection {
 
         let (to_node, from_lavalink) = mpsc::unbounded_channel();
         let (to_lavalink, from_node) = mpsc::unbounded_channel();
+        let lavalink_http = HyperClient::builder(TokioExecutor::new())
+            .http2_only(cfg!(feature = "lavalink-protocol-http2"))
```

Comment:
We should support both http/1 and http/2 if the `http2` feature is enabled (I think this is the default).